perun-core/src/main/resources/postgresChangelog.txt

```diff
@@ -6,6 +6,19 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.1.95
+ALTER TABLE application_reserved_logins ADD COLUMN user_id integer;
+ALTER TABLE application_reserved_logins ADD COLUMN extsourcename varchar;
+UPDATE application_reserved_logins arl SET user_id = a.user_id FROM application a WHERE arl.app_id = a.id;
```

Comment:
`user_id` will be probably null in most cases when migrating data, but I guess its ok, because of the way you select reserved logins.